CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,9 @@
+[cygnus-ngsi][Docker] Add support to run cygnus docker in multiagent mode (mysql, mongo, ckan, hdfs, postgresql and carto sinks) [#1486]
+[cygnus-ngsi][Docker] Upgrade to just enable sinks and channels depending on ENV vars [#1482]
+[cygnus-ngsi][Docker] Upgrade to CentOS 7, Maven 3.5.2 and Java 1.8.0 in Dockerfile
+[cygnus-common][Docker] Upgrade to CentOS 7, Maven 3.5.2 and Java 1.8.0 in Dockerfile
+[cygnus-twitter][Docker] Upgrade to CentOS 7, Maven 3.5.2 and Java 1.8.0 in Dockerfile
+[cygnus-ngsi][NGSINameMappingsInterceptor] Allow regular expression replacement for EntityIds (#1456)
+cygnus-ngsi][HttpBackend] Add support to recover data from response header (#1455)
+[cygnus-ngsi][KeystoneUtils] Create new Auth functionality (#1471)
+[cygnus-ngsi] [NGSIDynamoDBSink] [bug #1448] Southeast regions are bad coded (PR #1499)
```

Comment:
The file should include only the last line (all the other come for an old version of master branch). In addition, please use the following:

```
[cygnus-ngsi] [NGSIDynamoDBSink] [bug] Southeast regions are bad coded (#1448)
```

i.e. mentining the issue number, but not the PR number

Thanks!

Reply:
Thanks!! Fixed in f07d2ca